torch/onnx/_internal/diagnostics/infra/sarif_om/__init__.py

```diff
@@ -0,0 +1,107 @@
+# This file was generated by jschema_to_python version 1.2.3.
```

Comment:
Possible to say "Do not edit" in these files to make it clear?

Reply:
That will be further hacking the jschema_to_python tool, but why not 🤣 Changes will appear in the next PR then.